Packages/Core/Runtime/Events/AtomEvent.cs

```diff
@@ -31,10 +31,15 @@ public class AtomEvent<T> : AtomEventBase
         [SerializeField]
         [Range(0, 10)]
         [Tooltip("The number of old values (between 0-10) being replayed when someone subscribes to this Event.")]
-        private int _replayBufferSize = 1;
+        private int _replayBufferSize;
 
         private Queue<T> _replayBuffer = new Queue<T>();
 
+        private void OnEnable()
+        {
+            _replayBufferSize = AtomPreferences.ReplayBufferSize;
```

Comment:
This will just overwrite the replay buffer size if set from the Inspector. Not 100% sure how to do this properly though.